requirements.txt

```diff
@@ -1,19 +1,24 @@
 boto3==1.17.33
 botocore==1.20.33
 click==7.1.2
+decorator==5.0.9
 Flask==1.1.2
-Flask-CORS==3.0.10
+Flask-Cors==3.0.10
 Flask-SQLAlchemy==2.4.4
 itsdangerous==1.1.0
 Jinja2==2.11.3
 jmespath==0.10.0
 lxml==4.6.3
 MarkupSafe==1.1.1
+pbr==5.6.0
```

Comment:
Do we absolutely need pbr? ;-) Bad memories of debugging issues with it...

Reply:
yes, for packaging.